src/helpers/parseArgs.js

```diff
@@ -34,24 +36,24 @@ export function parseArgs(allArgs) {
     const isRelease = releaseRe.exec(arg)
     const isOpt = optionRe.exec(arg)
     if (arg === '--check-dependencies' || arg === '--fix-dependencies') {
-      builtin = builtinSignals['checkDependencies']
+      builtin = builtinSequences['checkDependencies']
       cmdArgs.push(arg)
     } else if (isRelease) {
       const type = isRelease[2]
-      const signal = `${type || 'default'}Release`
-      builtin = builtinSignals[signal]
+      const sequence = `${type || 'default'}Release`
+      builtin = builtinSequences[sequence]
       if (!builtin) {
         throw new Error(
-          `Invalid option '${arg}' (unknown builtin signal '${signal}').`
+          `Invalid option '${arg}' (unknown builtin signal '${sequence}').`
```

Comment:
Why you use `signal` here not `sequence`? ("unknown builtin signal")

Reply:
Nice catch !!